EG23.tex

```diff
@@ -465,45 +500,51 @@ \subsection{Case 1: Greedy Separations}
 replacement of higher breeding ratio SFRs with lower breeding ratio reactors
 the MOX spent fuel generation rate decreases sufficiently for reprocessing to
 "catch-up".
-
-As shown in the Figures \ref{fig:FC_Z} and \ref{fig:WR_Z}, the quantity of
-plutonium in storage follows the variation in the amount of MOX1 fuel in
-storage.
-
-\begin{figure}[h!]
-    \centering
-    \subfigure[Storage Fuel composition (zoomed)\label{fig:FC_Z}]		{\epsfig{figure=img/FuelInStorage_1_zoom,width=0.48\textwidth}}
-    \subfigure[Fuel waiting for reprocessing (zoomed)\label{fig:WR_Z}]	{\epsfig{figure=img/UNFWaitingReprocessing_1_zoom,width=0.48\textwidth}}
-    \caption{Plutonium in storage.\label{fig:FC_WR_zoom} }
-\end{figure}
+As shown in the Figure \ref{fig:storagecompo_1}, the reprocessing capacities of
+the LWR fuel is highier than the demand on plutonium to fabricate the MOX fuel,
+creating an accumulation of plutonium prior to 2100. After 2100, with the
+decommision of the LWR, less LWR used fuel is produces, allowing the reduction of
+the inventory of plutonium. In the other side, the reprocessing capacities of the
+FBR used fuel are well adapted to the demand: the plutonium storage experience a
+small accumulation just before the SFR switch of breeding ratio.
+ 
```

Comment:
I remove the zoomed part since it was not relevant anymore.
I add the comment about the new Fuel Ressource in Storage.